### ISSUE TITLE
Need for shorter tests

### ISSUE THREAD

**Issue by Dundalia** [rubynor/bigfive-web#138]
In order to make the test more versatile I believe it would be useful to implement **shorter** versions of the test. In particular, a shared limitation of existing psychometric measures is their length, demanding time from researchers and participants, which might cause boredom or fatigue, biasing the final results. 
There are scientific proofs of the quality and applicability of shorter tests, with a length varying from 10 to 44.

- [Evaluating the complete (44-item), short (20-item) and ultra-short (10-item) versions of the Big Five Inventory (BFI) in the Brazilian population](https://www.nature.com/articles/s41598-023-34504-1)
- [A short version of the Big Five inventory](https://journal.sipsych.org/index.php/IJP/article/download/1312/1050/5071)

**Comment by FalkenkralleMc** [comment]
yeah but a longer test would make it more accurate i really like it so
